x-pack/plugins/code/server/lsp/java_launcher.ts

```diff
@@ -118,6 +118,10 @@ export class JavaLauncher implements ILanguageServerLauncher {
 
     chmodSync(path.dirname(javaPath), '755');
 
+    process.env.CLIENT_HOST = '127.0.0.1';
+    process.env.CLIENT_PORT = port.toString();
```

Comment:
you create a new map instead of override parent process's map

Reply:
done